test/script/Deploy.s.t.sol

```diff
@@ -53,17 +62,19 @@ contract DeployTest is Test {
             keccak256(
                 abi.encodePacked(
                     type(AccountFactory).creationCode,
-                    abi.encode(address(_entryPoint), _accountImpl, _singleSignerValidationModule, _owner)
+                    abi.encode(address(_entryPoint), _accountImpl, _smaImpl, _singleSignerValidationModule, _owner)
                 )
             ),
             CREATE2_FACTORY
         );
 
         vm.setEnv("ACCOUNT_IMPL", vm.toString(address(_accountImpl)));
+        vm.setEnv("SMA_IMPL", vm.toString(address(_smaImpl)));
         vm.setEnv("FACTORY", vm.toString(address(_factory)));
         vm.setEnv("SINGLE_SIGNER_VALIDATION_MODULE", vm.toString(_singleSignerValidationModule));
 
         vm.setEnv("ACCOUNT_IMPL_SALT", vm.toString(uint256(0)));
+        vm.setEnv("SMA_IMPL_SALT", vm.toString(uint256(0)));
```

Comment:
Could you add the assertions for codesize and `runtimeCode` for this new contract, too?

Reply:
Codesize, good catch! RuntimeCode is a little more complicated (like the normal modular account) because of immutables-- do you think it's worth, in the test, deploying both account impls and comparing bytecode to have this check?

Reply:
Oh shoot, right, I think that's why I didn't do it for the regular account impl. Nevermind then, we should be fine with just the codesize.